src/main/java/io/aiven/kafka/connect/common/config/AivenCommonConfig.java

```diff
@@ -82,6 +84,30 @@ protected static void addOutputFieldsFormatConfigGroup(final ConfigDef configDef
             FORMAT_OUTPUT_FIELDS_VALUE_ENCODING_CONFIG,
             FixedSetRecommender.ofSupportedValues(OutputFieldEncodingType.names())
         );
+
+        addFormatTypeConfig(configDef, formatGroupCounter);
+    }
+
+    protected static void addFormatTypeConfig(final ConfigDef configDef,
```

Comment:
I wonder if it makes much sense to extract this to a method...

Reply:
It seems unusual, by GCS Sink doesn't not `useaddOutputFieldsFormatConfigGroup` method. It is used its own validator and another functions for this purpose. So, I wonder if I also should duplicate `addFormatTypeConfig` to both parts?

Reply:
- [ ] ????? Find out how to reuse common parts better

Reply:
The extracting configuration into Commons is a bit underbaked. Feel free to shape it :)

Reply:
If you feel it's possible to unify this between GCS and S3 now, let's do it. Otherwise, let's figure out what might be an obstacle.

Reply:
OK)))